new/LIT6552MalkEstifanos.xml

```diff
@@ -0,0 +1,57 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6552MalkEstifanos" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">መልክአ፡ እስጢፋኖስ፡</title>
```

Comment:
Nos. 43, 253, 316 in Chaine's Repertoire are for St. Stephen, maybe it's one of them?

Reply:
unfortunately EMML217 is not online... cannot check...